open-data/Lesson6-Sharing&Reuse.md

```diff
@@ -0,0 +1,311 @@
+
+Some core descriptors to look for are:
+
+* Who created this data?
+* When was it published?
+* How was the data collected?
+* What are the tools used for processing and handling the data?
+* What are the terms for using it?
+* What are the disciplines that used the data?
+* What are the communities that have developed around it?
+
+
+Ideally a fully documented Data Reuse plan should accompany the datasets providing detailed information on What, Who, Where, When and How, see also 5.3 Documenting your Data (Metadata)and Data Reuse Planning Template by mozilla open science
+
+[https://github.com/mozillascience/working-open-workshop/blob/8931bf19bedb99166b931f82dbd2fba815dc6fe1/handouts/data_reuse_plan_template.md](https://github.com/mozillascience/working-open-workshop/blob/8931bf19bedb99166b931f82dbd2fba815dc6fe1/handouts/data_reuse_plan_template.md)
+
+## Formats
+As noted previously, the format of data is key for its interoperability. Data formats are in two classes. Closed, proprietary formats are those where the specification for the way the data is stored is not publicly available and hence is tied explicitly to the software (or hardware) developed by an organization. Open formats have a specification that is available to all (though some may be proprietary and hence subject to conditions that will vary on the license terms applied to them).  Open data formats are cataloged in FAIRsharing.org.
+
+The format of a file should be described in the metadata associated with the data set. If metadata is not available then inspecting the file name (i.e. the postfix to the file name that comes after the period) can give an indication of what the file type is. Opening the file with software for a specific data format provides a similar test.
+
+If the format isn’t clear or is using a closed format where there isn’t software to access the data it is worth contacting the authors of the data set to see if they can help though the authors are not obliged to reply to you. Reaching out to the communities outlined in lesson 5 can also provide advice.
+
+## Tools
+
+Many of the details here are discussed in the Open Tools lesson. In the first instance in order to ensure that data is being read properly it is useful to check if a previously published analysis of the data can be reproduced. Papers outlining the analysis  should provide some description of the software used and the necessary steps in the analysis. In reproducing such an analysis one can determine if proprietary software is necessary to use the data and if you have all the necessary software or indeed source code to reproduce the results. If the results can be reproduced then you can be more confident in reusing the data. As described above, contacting the authors if there is an issue or likewise contacting communities described in lesson 5 can also help here. 
```

Comment:
```suggestion
Many of the details here are discussed in the Open Tools lesson. 
To ensure that you can reuse data you can check if a previously published analysis of the data can be reproduced. 
In reproducing such an analysis one can determine if you have all the necessary software, source code, and documentation needed to reproduce the results. 
If the results can be reproduced then you can be more confident in reusing the data. 
As described above, you can contact the authors if there is an issue or contact the communities described in lesson 5. 
```

Reply:
I find this paragraph a bit confusing: why is the title Tools? I think this is more about data quality?